src/group/GroupView.spec.ts

```diff
@@ -0,0 +1,36 @@
+import * as chai from "chai";
+import { GroupView } from "./GroupView";
+
+describe("GroupView", () => {
+    const view = new GroupView(
+        {
+            1: { id: 1, name: "org", scheme_id: 1},
+            2: { id: 2, name: "org2", scheme_id: 2}
+        }
+    );
+
+    it("creates a JSON view from a model", async () => {
+        const actual = await view.create({}, {
+            id: 1,
+            name: "group",
+            organisation_id: 2
+        });
+
+        chai.expect(actual.id).to.equal("/group/" + 1);
+        chai.expect(actual.name).to.equal("group");
+        chai.expect(actual.organisation).to.equal("/organisation/" + 2);
+    });
+
+    it("populates the links object", async () => {
```

Comment:
I think we need an extra test here - what shape are the organisation objects in? They should be OrganisationJsonView which means your GroupView will need to take a OrganisationView to create it, fun fun